subsys/settings/src/settings.c

```diff
@@ -18,6 +18,7 @@ sys_slist_t settings_handlers;
 
 static u8_t settings_cmd_inited;
 
+struct settings_handler *settings_handler_lookup(char *name);
```

Comment:
Make this static  and  definition as well (as already should be static, but is not).